objects_tdd/spec/lib/television_spec.rb

```diff
@@ -0,0 +1,108 @@
+require './lib/television'
+
+describe Television do
+  it 'can be created' do
+    expect(subject).to be_a(Television)
+  end
+
+  describe '#on?' do
+    it 'returns false by default' do
+      expect(subject.on?).to eq(false)
+    end
+  end
+
+  describe '#power' do
+    it 'toggles power on and off' do
+      subject.power
+      expect(subject.on?).to eq(true)
+      subject.power
+      expect(subject.on?).to eq(false)
+    end
+  end
+
+  describe '#set_channel(number)' do
+    it 'sets the channel to the number if within valid range' do
+      subject.power
```

Comment:
If you need to turn the power on before changing the channel, that's a good use for a `before` block:

    before(:each) do
      subject.power
    end

Reply:
@jasonnoble There is one test that should not have the power turned on.  How do you exclude a single test from running the before block?